include/tdi/arch/tna/tna_defs.h

```diff
@@ -50,9 +50,14 @@ typedef enum {
  * @brief Key Field Match Type. A key can have multiple fields,
  * each with a different match type
  */
-enum tdi_match_type_tna_e {
-  TDI_MATCH_TYPE_TNA_RANGE = TDI_MATCH_TYPE_ARCH,
-  TDI_MATCH_TYPE_TNA_ATCAM,
+enum tdi_tna_match_type_e {
+  TDI_TNA_MATCH_TYPE_RANGE = TDI_MATCH_TYPE_ARCH,
```

Comment:
The tdi_match_type_core_e as we discussed early, have the TDI_MATCH_TYPE_RANGE defined. Not sure TNA can use it or not. Here is the latest list.
 41 enum tdi_match_type_core_e {
 42   TDI_MATCH_TYPE_EXACT = TDI_MATCH_TYPE_CORE,
 43   TDI_MATCH_TYPE_TERNARY,
 44   TDI_MATCH_TYPE_RANGE,
 45   TDI_MATCH_TYPE_LPM,
 46 };

Reply:
That looks good. You can integrate that as part of your PR itself

Reply:
ok thanks